test/libsolidity/syntaxTests/inline_arrays/inline_array_fixed_types.sol

```diff
@@ -4,6 +4,4 @@ contract test {
     }
 }
 // ----
-// UnimplementedFeatureError: Not yet implemented - FixedPointType.
-// Warning 2072: (50-67): Unused local variable.
-// Warning 2018: (20-119): Function state mutability can be restricted to pure
+// UnimplementedFeatureError: Not yet Implemented - FixedPointType.
```

Comment:
Throwing for `fixed` has the side effect that analysis that may have otherwise continued is interrupted and hence other warnings/errors are not displayed to the user. Is this a breaking change?